development-vm/README.md

```diff
@@ -1,276 +1,212 @@
 
-```
-sudo rm /var/lib/mongodb/mongod.lock
-sudo service mongodb start
-```
+Since many of our apps depend on other apps, we normally run them using [bowler](https://github.com/JordanHatch/bowler) instead of foreman.
 
-### Errors with Vagrant 1.8.7
+To run particular apps with bowler, use:
 
-If you use Vagrant 1.8.7 you may have this problem:
+    dev$ bowl content-tagger
 
-```
-➜  development-vm git:(master) vagrant up
-installing vagrant-dns plugin is recommended
-Bringing machine 'default' up with 'virtualbox' provider...
-==> default: Box 'govuk_dev_trusty64_20160323' could not be found. Attempting to find and install...
-    default: Box Provider: virtualbox
-    default: Box Version: >= 0
-==> default: Box file was not detected as metadata. Adding it directly...
-==> default: Adding box 'govuk_dev_trusty64_20160323' (v0) for provider: virtualbox
-    default: Downloading: https://govuk-dev-boxes-test.s3.amazonaws.com/govuk_dev_trusty64_20160323.box
-An error occurred while downloading the remote file. The error
-message, if any, is reproduced below. Please fix this error and try
-again.
-```
+This will also run all of the dependencies defined in the Pinfile.
 
-it looks like a problem with this specific version of Vagrant. Using version 1.8.6 works instead: https://releases.hashicorp.com/vagrant/1.8.6/
-You can find more information about this issue here: https://github.com/mitchellh/vagrant/issues/8002
+If you don't need an optional dependency, you can pass the `-w` option:
 
-#### `librarian:install` fails due to permission errors
+    dev$ bowl whitehall -w mapit
 
-Seeing `chown` / `OperationNotPermitted` errors during the `librarian:install` rake task?
+## 9. Keep your VM up to date
 
-Try `vagrant provision` on your host machine, as above.
+There are a few scripts that should be run regularly to keep your VM up to date. In `govuk-puppet/development-vm` there is `update-git.sh` and `update-bundler.sh` to help with this. Also, `govuk_puppet` should be run from anywhere on the VM regularly.
 
-#### Hostname provisioning errors
+The following invocation will do all of this for you.
 
-If you use Vagrant v1.8.x, you may encounter an error along these lines during provisioning:
+    dev$ cd /var/govuk/govuk-puppet/development-vm
+    dev$ ./update-all.sh
 
-```
-Default: Setting hostname...
-/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/guests/ubuntu/cap/change_host_name.rb:37:in `block in init_package': unexpected return (LocalJumpError)
-    from /opt/vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/communicators/ssh/communicator.rb:222:in `call’
-```
+This will run:
 
-Updating to the latest version (v1.8.5+) might resolve this error. If not, try `sudo vim`ing (or `sudo nano`, whichever you prefer) into that file and removing the offending `return` line.
+1. `git pull` on each of the applications checked out in `/var/govuk`
```

Comment:
These look like a numbered list of items, but they'd probably work better as bullets.